TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -43,10 +44,15 @@ object TeleOpMovement : Component {
         var joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
         // if the right bumper is pressed move slower
-        if (gamepad.right_bumper) {
+        if (gamepad.left_bumper) {
             joyMagnitude *= SLOW_MULTIPLIER
         }
 
+        // OutTake drop
+        if (gamepad.left_bumper) {
+            OutTake.drop()
+        }
```

Comment:
Did you mean to change this to `left_bumper`? Pressing it will apply both the slow multiplier and also drop the outtake.

Reply:
I did mean to do that!